test/bigtest/network/config.js

```diff
@@ -23,7 +35,17 @@ export default function config() {
     // };
   });
 
-  this.put(`${ORDERS_API}/:id`, noop);
+  this.put(`${ORDERS_API}/:id`, (schema, request) => {
+    const attrs = JSON.parse(request.requestBody);
+
+    return schema.orders.create(attrs);
+  });
+
+  this.post(ORDERS_API, (schema, request) => {
```

Comment:
the same question as above about shortcut. I don't see any logic implemented, so shortcut method might work